wrangling.Rmd

```diff
@@ -1417,34 +1390,30 @@ region_lang |>
   map_dfc(max)
 ```
 
-Which `map` function you choose depends on what you want to do with the
-output; you don't always have to pick `map_dfc`!
-
 > **Note on calculating summary statistics with `map` when there are `NA`s**:
 > 
 > Similarly to when we use base R statistical summary functions 
 > (e.g., `max`, `mix`, `mean`, `sum`, etc) with `summarize`, 
 > `map` functions paired with base R statistical summary functions
-> also return `NA`s when we apply them to columns that 
-> contain `NA`s in the data frame.
+> also return `NA` values when we apply them to columns that 
+> contain `NA` values. \index{missing data}
 > 
 > To avoid this, again we need to add the argument `na.rm = TRUE`.
 > When we use this with `map` we do this by adding a `,` 
-> and then `na.rm = TRUE`, 
-> after specifying the function we want `map` to apply, as illustrated below:
+> and then `na.rm = TRUE` after specifying the function, as illustrated below:
 > 
 > ``` {r}
 > region_lang_na |>
 >   select(mother_tongue:lang_known) |>
 >   map_dfc(max, na.rm = TRUE)
 > ```
 
-The `map` family functions are generally quite useful for solving many problems 
+The `map` functions are generally quite useful for solving many problems 
 involving repeatedly applying functions in R. 
 Additionally, their use is not limited to columns of a data frame;
 `map` family functions can be used to apply functions to elements of a vector,
-or a list, and even to lists of data frames, or nested data frames.
-To learn more about them see the resource linked to in the Additional Resources
+or a list, and even to lists of (nested!) data frames.
+To learn more about the `map` functions, see the additional resources
 section at the end of this chapter.
 
 ## Apply functions across many columns with `mutate` and `across`
```

Comment:
below this it says "This is a common task when converting units of measurement across many columns, and when standardizing data before modelling" 

I don't think we should talk about standardizing here since we haven't introduced readers to this concept yet

Reply:
Also below the cartoon -- again I think it would be useful to add numbers to the cartoon.

Reply:
I could see how the cartoons would work nicely if they were used side by side to compare the concepts so students can see how each of the functions differ. But I don't know if they work without more context when introducing it as the first intro to a new concept

Reply:
agree to first comment, and fixed.

for the second/third, i've made an issue -- punted